{
	"id": "CVE-2017-15128",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2017-15128",
				"description": "A flaw was found in the hugetlb_mcopy_atomic_pte function in mm/hugetlb.c in the Linux kernel before 4.13.12. A lack of size check could cause a denial of service (BUG).",
				"cwe": [
					{
						"source": "redhat",
						"cwe": [
							"CWE-119"
						]
					}
				],
				"references": [
					{
						"source": "CVE",
						"url": "http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=1e3921471354244f70fe268586ff94a97a6dd4df"
					},
					{
						"source": "CVE",
						"url": "https://access.redhat.com/security/cve/CVE-2017-15128"
					},
					{
						"source": "CVE",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1525222"
					},
					{
						"source": "CVE",
						"url": "https://github.com/torvalds/linux/commit/1e3921471354244f70fe268586ff94a97a6dd4df"
					},
					{
						"source": "CVE",
						"url": "https://www.kernel.org/pub/linux/kernel/v4.x/ChangeLog-4.13.12"
					},
					{
						"source": "redhat",
						"url": "http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=1e3921471354244f70fe268586ff94a97a6dd4df"
					},
					{
						"source": "redhat",
						"url": "https://access.redhat.com/security/cve/CVE-2017-15128"
					},
					{
						"source": "redhat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1525222"
					},
					{
						"source": "redhat",
						"url": "https://github.com/torvalds/linux/commit/1e3921471354244f70fe268586ff94a97a6dd4df"
					},
					{
						"source": "redhat",
						"url": "https://www.kernel.org/pub/linux/kernel/v4.x/ChangeLog-4.13.12"
					}
				],
				"modified": "2024-08-05T19:50:16.308Z"
			}
		}
	],
	"data_source": {
		"id": "mitre-v5",
		"raws": [
			"vuls-data-raw-mitre-v5/2017/CVE-2017-15128.json"
		]
	}
}
